### PR TITLE
[CID-486] - Add ?apply_rcn_roles support to v2.0 List Tenants

### PR DESCRIPTION
**Pull request by kidster**
**Quality audit reminder**

Doc team: While logged in using your O365 account, complete the quality checklist at http://bit.ly/2hwvBKX. Use the checklist for significant doc changes or additions submitted by you or a non-doc staff contributor.

### CODE REVIEW

File: api-docs/api-reference/methods/tenants/get-list-tenants-v2.0.rst [hunk 1]
```diff
@@ -7,12 +7,18 @@ List tenants
...
     GET /v2.0/tenants
 
-Get a list of tenants, or look up a tenant by name.
+Get a list of tenants.
 
-Identity administrators can use this operation to retrieve a list of tenants
-or an individual tenant by name.
+The list of tenants returned is determined by the X-Auth-Token provided on the
+request. The list will include tenants a user has access to. The tenants the
+user has access to via RCN roles will only be returned if requested.
```
Comment (catlook) [inline api-docs/api-reference/methods/tenants/get-list-tenants-v2.0.rst]: Replace lines 12-14 with the following:
The list of tenants returned includes only those to which the user has access, based on the user's X-Auth-token. The tenants to which the user has access via RCN roles are returned only if requested.

File: api-docs/api-reference/methods/tenants/get-list-tenants-v2.0.rst [hunk 2]
```diff
@@ -7,12 +7,18 @@ List tenants
...
 
+.. note::
 
+    - The list tenants operation can be made by any users with an Identity
+      role.
+    - Tenants associated to a user via an RCN role will **ONLY** be returned
```
Comment (catlook) [inline api-docs/api-reference/methods/tenants/get-list-tenants-v2.0.rst]: ... role will be returned **only**...

Comment (kidster) [inline api-docs/api-reference/methods/tenants/get-list-tenants-v2.0.rst]: done

File: api-docs/api-reference/methods/tenants/get-list-tenants-v2.0.rst (hunk 1 above)
Comment (kidster) [inline api-docs/api-reference/methods/tenants/get-list-tenants-v2.0.rst]: done

**Pull request merged**